src/components/InfoUtsattFristJuletid.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Alert } from "@navikt/ds-react";
+import { getForhandsvarselFrist } from "@/utils/forhandsvarselUtils";
+import { addWeeks } from "@/utils/datoUtils";
+
+const texts = {
+  infoText:
+    "Fristen i brevet er automatisk flyttet fra 3 til 6 uker i forbindelse med jul og røde dager. Du vil få en oppgave i Modia syfo etter 6 uker. Etter nyttår endres fristen tilbake til 3 uker.",
```

Comment:
Litt presisering på ordlyden: De får opp personen i oversikten med en gang vi sender forhåndsvarsel, men det kommer rød prikk i Modia etter at fristen er gått ut. Så det spørs kanskje litt i hva vi tenker på å "få en oppgave i Modia Syfo" betyr for oss (og dem).

Og vil fristen flyttes tilbake til 3 uker ved nyttår eller 17. desember? Hvis vi vet akkurat når, så tenker jeg vi kan skrive det 🤷🏼‍♂️

Vet at teksten er kopiert fra forrige løsning, men vi kan se på det. Kanskje Stine har noen tanker. Vi kunne evt brukt noe av teksten som ligger i [Yammer-innlegget](https://engage.cloud.microsoft/main/org/nav.no/threads/eyJfdHlwZSI6IlRocmVhZCIsImlkIjoiMzA4MjY0MjE4MzE2ODAwMCJ9?trk_copy_link=V2).

Reply:
Ja godt poeng! Snakket med Stine om teksten. Hvis vi skal endre til at det står eksakt frist så burde vi kanskje endre datoen til å være litt senere slik at bruker får flere arbeidsdager etter nyttår til å feks dra til legen. Tekstforslag:
`Fristen i brevet er automatisk flyttet fra 3 til 6 uker i forbindelse med jul og røde dager. Den 19. desember endres fristen automatisk tilbake til 3 uker. `